File: Python/Minor_Assignment_3/Q5.py
def countDigits(n):
    count=0
    while n!=0:
        count+=1
        n=n//10
    return count

n=int(input("Enter a number:"))
print("Number of digits in the number:",countDigits(n))

'''
Output--
Enter a number:6887192
Number of digits in the number: 7
'''

File: PPWC/Minor_Assignment_6/Q5.c
#include<stdio.h>
void f(int n){
    if(n<=1){
        printf("%d",n);
    }
    else{
        f(n/2);
        printf("%d",n%2);
    }
}
int main(){
    f(173);
    return 0;
}

/*
Output--
10101101
*/